cernopendata/modules/fixtures/data/docs/cern-open-data-release-2024/cern-open-data-release-2024.md

```diff
@@ -0,0 +1,44 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024. The changes were deployed first in the [CERN Open Data QA instance](https://opendata-qa.cern.ch), and then propagated to the production instance.
+
+The list of changes include:
+
+* Change of technology, from Angular to ReactJS and Bootstrap[3,4] to Semantic UI for the web interface
+* Update the dependencies on the invenio framework modules
+* Move to xrootd version 5
+* Move to docker compose version 2
+* Change the self-maintained ElasticSearch instance by OpenSearch instances provided by the CERN OpenSearch service
```

Comment:
I would phrase the ES to OS change again to be more user-centric. Data scientists do not really care about underlying search technology and where it is hosted, they care more that this change is improving some search nuisances that they might have had with the old system. So I would rather highlight something like:

- Improvements to the search functionality making the returned query results to be more relevant.
- Improvements to the faceting functionality allowing to combine many new facet categories together.

I guess you get what I mean?

Reply:
Ok, rephrased pointing out that the version that we were running was not supported anymore.